component/src/main/java/io/siddhi/extension/io/cdc/source/polling/CDCPoller.java

```diff
@@ -92,7 +97,7 @@ public CDCPoller(String url, String username, String password, String tableName,
         } else {
             log.debug(DefaultPollingStrategy.class + " is selected as the polling strategy.");
```

Comment:
Shall we validate and throw app creation error if cron and some other strategy is selected?

Reply:
fixed in ee838f87eb853dd4cccb33250c2fa8f7d620a3cc